tests/unit/serve/runtimes/worker/test_worker_runtime.py

```diff
@@ -231,7 +232,7 @@ def start_runtime(args, cancel_event):
             try:
                 s.connect((args.host, args.port))
                 connected = True
-            except ConnectionRefusedError:
+            except:
```

Comment:
On my Mac the error thrown is different than the single expected error. This was a blocker for running the tests in the `worker` module so I removed it because the exception itself was not used for asserts.